Load ruby ftplugin in slim ftplugin

lisinge opened the pull request:
This will load mappings from vim-rails and co.
This will give you access to mappings from `vim-rails` such as `gf` on partials.
I'm not a professional vimscripter but i think this is the right way of doing it.

Was told by timpope that this should be done here in: https://github.com/tpope/vim-rails/issues/524

tpope commented:
This won't work because the Ruby ftplugin will bail when it sees `b:did_ftplugin` set. Also you'll need to combine the Ruby `b:undo_ftplugin` with the Slim one. I'd recommend starting with the Haml ftplugin and deleting everything you don't care about.

lisinge commented:
Reworked and rebased after @tpope's comment, if you have the time to throw a quick peek at this i would very much appreciate it @tpope.
Hopefully this properly handles `b:undo_ftplugin`.

tpope commented on ftplugin/slim.vim:
```diff
@@ -1,9 +1,25 @@
+let s:undo_ftplugin = ""
+
+" Override our defaults if these were set by an included ftplugin.
+if exists("b:undo_ftplugin")
```
This was for the HTML ftplugin and does nothing if it's not loaded. You can drop all the `s:undo_ftplugin`.

stonean commented:
Thank you @lisinge and @tpope
